doc/nrf/releases_and_maturity/release_notes.rst

```diff
@@ -19,6 +19,7 @@ This page is included only in the latest documentation, because it might contain
    :maxdepth: 1
    :caption: Subpages:
 
+   releases/release-notes-changelog
```

Comment:
line no 13 needs update as : _When looking at this latest documentation, be aware of the following aspects:_

and add the following bullet point after line no:14. Hence, have two bullet points.

_* Changes between releases are tracked on the :ref:`ncs_release_notes_changelog` page, but the main branch might contain additional changes that are not listed on that page._